### ISSUE TITLE
can't load svgs with a capital letter in the name 

### ISSUE THREAD

**Issue by wal0x** [cascornelissen/svg-spritemap-webpack-plugin#84]
@cascornelissen  I am using webpack@4.28.4 on a VueJS 2.6.10 project, and "svg-spritemap-webpack-plugin": "3.3.7".
The svg with the name a003_test_a.svg works but not 003_test_a.svg nor a003_Test_a.svg for instance.

**Comment by cascornelissen** [comment]
@wal0x, do you have more information? The examples literally use filenames like `1.svg` and they seem to be working. What versions of the plugin and webpack are you using, what options do you pass to the plugin, etc.?

**Comment by wal0x** [comment]
@cascornelissen I've verified again, and I found I was rushing in fact the bug is only in svg with a capital letter somehere in the name. (noting to do with numbers, I'll edit the issue)

**Comment by wal0x** [comment]
I just found the bug, it's just the name of the svg is lower-cased. Therefor to use A.svg I had to link "#sprite-a" not "#sprite-A"

**Comment by cascornelissen** [comment]
Yeah, that's correct. The plugin uses [`idify`](http://npmjs.com/package/idify) which calls `.toLowerCase()` on the input 🤔 Probably because uppercase characters are [not allowed in id until HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id).

> Using characters except ASCII letters, digits, '_', '-' and '.' may cause compatibility problems, as they weren't allowed in HTML 4. Though this restriction has been lifted in HTML5, an ID should start with a letter for compatibility.

I might make an option to pass a custom function te replace `idify` to solve the problem you're running into.

**Comment by cascornelissen** [comment]
Alright, I've introduced a new option `sprite.idify` which takes a function that will be used to transform the filename to a valid id. Could you try installing the plugin directly from the `master` branch by executing the following command:

```
npm install --save-dev cascornelissen/svg-spritemap-webpack-plugin#master
```

And then use the `sprite.idify` option to specify a custom function, for example:

```es6
new SVGSpritemapPlugin('src/**/*.svg', {
    sprite: {
        idify: (value) => value.replace(/[\s]+/g, '-') // Replace all whitespace with a dash
    }
})
```

Let me know! :v:

**Comment by wal0x** [comment]
Working 👍 

> Alright, I've introduced a new option `sprite.idify` which takes a function that will be used to transform the filename to a valid id. Could you try installing the plugin directly from the `master` branch by executing the following command:
> 
> ```
> npm install --save-dev cascornelissen/svg-spritemap-webpack-plugin#master
> ```
> 
> And then use the `sprite.idify` option to specify a custom function, for example:
> 
> ```js
> new SVGSpritemapPlugin('src/**/*.svg', {
>     sprite: {
>         idify: (value) => value.replace(/[\s]+/g, '-') // Replace all whitespace with a dash
>     }
> })
> ```
> 
> Let me know! ✌️

Working 👍

**Comment by cascornelissen** [comment]
Awesome, I'll add some tests, finish up the docs and release a new minor version later today 👍

**Comment by cascornelissen** [comment]
@wal0x, the fix for this issue has been released in version [`3.4.0`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.4.0) 🚀
